docs/CONTRIBUTING.md

```diff
@@ -41,44 +42,52 @@ Be sure to mention that you're using Connectors and also let us know what servic
 You can also find us in the `#enterprise-search` channel of the [Elastic Community Slack](http://elasticstack.slack.com).
 
 ## Types of contributions
-### Initial contribution
+
+### Customize a connector
+
+To customize an _existing_ connector, follow these steps:
+
+1. Customize the source file for your data source from [connectors/sources](../connectors/sources)
+2. Add a unit test in [connectors/sources/tests](../connectors/sources/tests) with **+90% coverage**
```

Comment:
Should we link to a testing reference, or explain how to validate your coverage metrics using `make test`?

Reply:
How about something like:
```
Test coverage is run as part of [unit tests](https://github.com/elastic/connectors/blob/main/docs/DEVELOPING.md#testing-the-connector). Look for your file at the end of the console output.
```
?